### ISSUE TITLE
doesn't work in proton

### ISSUE THREAD

**Issue by ashkitten** [rie-kumar/MusicVideoPlayer#29]
installed with QBeat, the ui loads but searching for a song doesn't work.

logfile: https://gist.github.com/ashkitten/e035779bf90572fec6d58458fa97dbb6

**Comment by b-rad15** [comment]
The first Error is that it isn't able to set the console output encoding but from there a lot of other stuff that I don't understand.

I'm not familiar with how proton does these things, but can you try running "Z:\home\ash\.local\share\Steam\steamapps\common\Beat Saber/Youtube-dl/youtube-dl.exe" "ytsearch:Connect (TV Size) - ClariS" and seeing if anything is downloaded?

**Comment by ashkitten** [comment]
yes, trying that downloads a video
```
WINEPREFIX=~/.steam/root/steamapps/compatdata/620980/pfx steam-run ~/.steam/root/steamapps/common/Proton\ 5.0/dist/bin/wine youtube-dl.exe "ytsearch:Connect (TV Size) - ClariS"
```

**Comment by b-rad15** [comment]
might work on the new release, I found a way to not need the exe at all

**Comment by ashkitten** [comment]
thanks, it now is able to search and download videos but it can't guess offset, cut, or preview them. it looks like some ffmpeg dll may be missing according to the log: https://gist.github.com/ashkitten/09d4e718a8077fa76f077a5ffbdd00e0

**Comment by b-rad15** [comment]
I don't think that it means the file wasn't found but that the stuff in the file couldn't be loaded since the assembly part is unknown.
If the file is in the folder Youtube-dl\SyncVideoWithAudio\ffmpeg32 then this may just be an issue with proton not being able to load that native library

**Comment by ashkitten** [comment]
you appear to be right, but i don't know how to get it working... also, the issue that the video can't even be loaded in beat saber appears to be separate. it might be solved in a future update to proton (wine is getting a bunch of media foundation fixes), but beat saber doesn't seem to work with anything other than proton-5.0 atm so i can't test those changes
